src/azul/openapi/params.py

```diff
@@ -54,15 +54,18 @@ def query(name: str, type_: Union[TYPE, schema.optional], **kwargs: PrimitiveJSO
     return _make_param(name, in_='query', type_=type_, **kwargs)
 
 
-def _make_param(name: str, in_: str, type_: Union[TYPE, schema.optional], **kwargs: PrimitiveJSON) -> JSON:
+def _make_param(name: str, in_: str, type_: Union[TYPE, schema.optional], **kwargs: AnyJSON) -> JSON:
     is_optional = isinstance(type_, schema.optional)
     if is_optional:
         type_ = type_.type_
+    type_property = (schema.wrap_json_schema_content(kwargs.pop('content'))
```

Comment:
This is too much hidden and brittle magic for my taste. Popping the `content` key only to then add it again seems awkward. The brittleness arises from the fact that the condition looks at the type but `if` branch then acts on the kwargs.